modules/atto/shared/src/main/scala/cron4s/atto/QuartzParser.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2017 Antonio Alonso Dominguez
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cron4s.atto
+
+import _root_.atto.{Parser => AttoParser, _}
+import atto.Atto._
+
+trait QuartzParser extends cron4s.parser.Parser with CronExprParser {
```

Comment:
I always tried to avoid an explicit reference to `Quartz` in the code (and user guide) because I mostly focused on this library being able to parse vanilla Unix cron. The limitations listed in the documentation (plus the lack of a Year field) were the way I chose to say said "This isn't Quartz-compatible" without actually mentioning "Quartz".

Makes me wonder if with this new "dialect-support" there should be a unix-specific dialect separate from the Quartz one... food for thought.